Assets/UnZENity-Core/Scripts/Vob/StationaryLight.cs

```diff
@@ -212,7 +215,11 @@ public static void InitStationaryLights()
             Shader.SetGlobalVectorArray(GlobalStationaryLightPositionsAndAttenuationShaderId,
                 lightPositionsAndAttenuation);
             Shader.SetGlobalVectorArray(GlobalStationaryLightColorsShaderId, lightColors);
-            _threadSafeLightData.Clear(); // Clear the thread safe data as it is no longer needed.
+        }
+
+        public static void ClearThreadSafeData()
```

Comment:
This function never gets called. Can you (e.g.) add it to InitStationaryLights? (Or simply clear the _threadSafeLightData there.)

Reply:
We can't clear it until World gets created - otherwise _StationaryLightCount inside Lit-World shader (which world chunks are using) won't be properly initialised. I'm calling this right after `WorldCreator.CreateAsync()`
```csharp
                // 4.
                if (config.EnableWorldMesh)
                {
                    // initialize Lights before world creation
                    vobRoot.SetActive(true); // temporary enable vobRoot
                    await StationaryLight.InitializeThreadSafeLightData();
                    vobRoot.SetActive(false); // disable to save some seconds in loading time ;p

                    await WorldCreator.CreateAsync(config, GameGlobals.Loading, worldRoot);
                    StationaryLight.ClearThreadSafeData(); // <-- here
                }
```